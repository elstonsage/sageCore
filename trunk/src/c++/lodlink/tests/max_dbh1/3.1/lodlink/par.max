
 LINKAGE ANALYSIS BETWEEN LOCUS ABO      AND LOCUS AK1     
 ==========================================================


 --------------------------------------------------------------------------------------------------------------------------------
 --------------------------------------------------------------------------------------------------------------------------------
 MAXFUN
 COPYRIGHT (C) 1997 CASE WESTERN RESERVE UNIVERSITY.
 
 TOTAL # PARAMETERS IN THE MODEL NT =  2; # PARAMETERS TO BE ESTIMATED NE =  2; # DEPENDENT PARAMETERS ND =  1
 MAXIMUM # ITERATIONS MAXIT = 999
 CONTROL VALUES:
    IXVC = 0; IHIT = 0;
    EPSD =  .100E-01; YOTA =  .169E-05; EPST =  .130E-01; EPSC1 =  .100E-01; EPSC2 =  .100E-02; EPSC3 =  .100E-02
 --------------------------------------------------------------------------------------------------------------------------------
0METHOD 2:  DIRECT SEARCH METHOD, WITHOUT 2**(NI)-TRIAL SEARCH
0INITIAL CONDITIONS
    PARAMETER                           LOWER BOUND      INITIAL EST      UPPER BOUND     IN STEP FACT       STATUS
     1                                    .00000000        .50000001E-01    .50000000        .50000001E-01   IND-FN, MAY VARY
     2                                    .00000000        .50000001E-01    .50000000        .50000001E-01   DEPENDENT       
    INITIAL FUNCTION VALUE  -.6234829281E+02
 --------------------------------------------------------------------------------------------------------------------------------
0BASIC ITERATION PROCESS STOPPED AFTER ITERATION   5 DUE TO NEGLIGIBLE DIFFERENCE IN COMPUTED FUNCTION VALUES     .5320332117E-03
 (CONVERGENCE CRITERION 3)
0PARAMETER  1 FIXED AT BOUND    .00000000    
0AFTER ITERATION   4, ALL INDEPENDENT PARAMETERS CONVERGED TO BOUNDS
0FINAL CONDITIONS
    PARAMETER                             FINAL EST          STD DEV      FIRST DERIV       STATUS
     1                                    .00000000       ----------       ----------       IND-FN, FIXED BY MAXFUN AT BOUND    
     2                                    .00000000       ----------       ----------       DEPENDENT                           
    FINAL FUNCTION VALUE  -.6233197311E+02
    NUMBER OF FUNCTION EVALUATIONS    2

 --------------------------------------------------------------------------------------------------------------------------------
 --------------------------------------------------------------------------------------------------------------------------------

 --------------------------------------------------------------------------------------------------------------------------------
 --------------------------------------------------------------------------------------------------------------------------------
 MAXFUN
 COPYRIGHT (C) 1997 CASE WESTERN RESERVE UNIVERSITY.
 
 TOTAL # PARAMETERS IN THE MODEL NT =  2; # PARAMETERS TO BE ESTIMATED NE =  2; # DEPENDENT PARAMETERS ND =  1
 MAXIMUM # ITERATIONS MAXIT = 999
 CONTROL VALUES:
    IXVC = 2; IHIT = 0;
    EPSD =  .100E-01; YOTA =  .169E-05; EPST =  .130E-01; EPSC1 =  .100E-01; EPSC2 =  .100E-02; EPSC3 =  .100E-02
 --------------------------------------------------------------------------------------------------------------------------------
0METHOD 5:  VARIABLE METRIC METHOD, USING NEGATIVE IDENTITY FOR INITIAL HESSIAN
0INITIAL CONDITIONS
    PARAMETER                           LOWER BOUND      INITIAL EST      UPPER BOUND       STATUS
     1                                    .00000000        .00000000        .50000000       IND-FN, MAY VARY
     2                                    .00000000        .00000000        .50000000       DEPENDENT       
    INITIAL FUNCTION VALUE  -.6233197311E+02
 --------------------------------------------------------------------------------------------------------------------------------
0PARAMETER  1 FIXED AT BOUND    .00000000    
0AFTER ITERATION   4, ALL INDEPENDENT PARAMETERS CONVERGED TO BOUNDS
