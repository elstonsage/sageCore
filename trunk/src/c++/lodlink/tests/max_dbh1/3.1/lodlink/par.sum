1S.A.G.E. RELEASE 3.1 -- LODLINK   		DECEMBER 1997
 COPYRIGHT (C) 1997 CASE WESTERN RESERVE UNIVERSITY.


 LODLINK GENERAL PARAMETERS
 --------------------------
     TITLE ............................... DBH data.  Lodlink parameter file.                                              
     NUMBER OF MARKER LOCI ...............    2
     FEMALE AND MALE SYMBOLS ............. F  M
     MORTON'S HOMOGENEITY TEST ........... NO 
     SMITH'S HOMOGENEITY TEST AND TEST
     FOR LINKAGE UNDER HETEROGENEITY ..... NO 
     NUMBER OF PEDIGREES .................    1
     RECOMBINATION FRACTION 
      -- FOR MALES, (THETA M) ............   .0000    .0100    .0500    .1000    .2000    .3000    .4000
      -- FOR FEMALES, (THETA F) ..........   .0000    .0100    .0500    .1000    .2000    .3000    .4000
1S.A.G.E. RELEASE 3.1 -- LODLINK   		DECEMBER 1997
 COPYRIGHT (C) 1997 CASE WESTERN RESERVE UNIVERSITY.


 LINKAGE ANALYSIS BETWEEN LOCUS ABO      AND LOCUS AK1     
 ==========================================================

                            -------------------------------------------------------------------
 THETA M                    !     .0000    .0100    .0500    .1000    .2000    .3000    .4000 !
 THETA F                    !     .0000    .0100    .0500    .1000    .2000    .3000    .4000 !
 ----------------------------------------------------------------------------------------------
 !   PEDIGREE   NUMBER OF   !                                                                 !
 !   NUMBER     INDIVIDUALS ! LOD SCORES                                                      !
 ----------------------------------------------------------------------------------------------
 !   00005         88       !     .0385    .0370    .0314    .0250    .0142    .0064    .0016 !
 ----------------------------------------------------------------------------------------------
 !                 88       !     .0385    .0370    .0314    .0250    .0142    .0064    .0016 !
 ==============================================================================================

                                         LOD      CHI             P-VALUE
                             ESTIMATE   SCORE    SQUARE  UPPER BOUND   APPROX. *
                             ========  =======  =======  ===========  ==========
 EQUIVALENT NUMBER OF MEIOSES    .128
 THETA IN (0, 1)                 .000     .039     .177      .915153     .336840 (ONE-SIDED)
 VARIANCE OF THETA            -------
 COMPLEMENT (1 - THETA)         1.000     .039

 * NOTE:  APPROXIMATE P-VALUES ASSUME A LARGE EQUIVALENT NUMBER OF
          INFORMATIVE MEIOSES.
