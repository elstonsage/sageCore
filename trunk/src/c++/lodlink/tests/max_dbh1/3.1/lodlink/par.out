1S.A.G.E. RELEASE 3.1 -- LODLINK   		DECEMBER 1997
 COPYRIGHT (C) 1997 CASE WESTERN RESERVE UNIVERSITY.


 LODLINK GENERAL PARAMETERS
 --------------------------
     TITLE ............................... DBH data.  Lodlink parameter file.                                              
     NUMBER OF MARKER LOCI ...............    2
     FEMALE AND MALE SYMBOLS ............. F  M
     MORTON'S HOMOGENEITY TEST ........... NO 
     SMITH'S HOMOGENEITY TEST AND TEST
     FOR LINKAGE UNDER HETEROGENEITY ..... NO 
     NUMBER OF PEDIGREES .................    1
     RECOMBINATION FRACTION 
      -- FOR MALES, (THETA M) ............   .0000    .0100    .0500    .1000    .2000    .3000    .4000
      -- FOR FEMALES, (THETA F) ..........   .0000    .0100    .0500    .1000    .2000    .3000    .4000

 Locus ABO     

 A1  =   .1904, A2  =   .0612, B   =   .0728, O   =   .6756, 
  No.   Genotype   Phenotype   Genotypic Frequency

    1   A1  A1       1                .036252
    2   A1  A2       1                .023305
    3   A1  B        2                .027722
    4   A1  O        1                .257268
    5   A2  A2       3                .003745
    6   A2  B        4                .008911
    7   A2  O        3                .082693
    8   B   B        5                .005300
    9   B   O        5                .098367
   10   O   O        6                .456435

 Locus AK1     

 1   =   .9620, 2   =   .0380, 
  No.   Genotype   Phenotype   Genotypic Frequency

    1   1   1        1                .925444
    2   1   2        2                .073112
    3   2   2        3                .001444

 LINKAGE ANALYSIS BETWEEN LOCUS ABO      AND LOCUS AK1     
 ==========================================================


 FAMILY NUMBER    1 (ID. IS 00005)
 ---------------------------------

 %LODLINK-I: There are   30 individuals with missing marker data.

         Rm       Rf     Log10[L(R)]     LOD-SCORE
      .0000    .0000      -27.070432       .038506
      .0100    .0100      -27.071900       .037038
      .0500    .0500      -27.077520       .031419
      .1000    .1000      -27.083951       .024988
      .2000    .2000      -27.094726       .014212
      .3000    .3000      -27.102569       .006369
      .4000    .4000      -27.107338       .001601
      .5000    .5000      -27.108938       .000000
